tests/benchmarks/benchmark-ewbg-c2hdm.cpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: 2021 Philipp Basler, Margarete Mühlleitner and Jonas
+// Müller
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include <benchmark/benchmark.h>
+
+#include <BSMPT/baryo_calculation/CalculateEtaInterface.h>
+#include <BSMPT/minimizer/Minimizer.h>
+#include <BSMPT/models/ClassPotentialOrigin.h> // for Class_Potential_Origin
+#include <BSMPT/models/IncludeAllModels.h>
+#include <BSMPT/models/ModelTestfunctions.h>
+
+#include "C2HDM.h"
+
+namespace
+{
+const std::vector<double> example_point_C2HDM{/* lambda_1 = */ 3.29771,
+                                              /* lambda_2 = */ 0.274365,
+                                              /* lambda_3 = */ 4.71019,
+                                              /* lambda_4 = */ -2.23056,
+                                              /* Re(lambda_5) = */ -2.43487,
+                                              /* Im(lambda_5) = */ 0.124948,
+                                              /* Re(m_{12}^2) = */ 2706.86,
+                                              /* tan(beta) = */ 4.64487,
+                                              /* Yukawa Type = */ 1};
+const Compare_C2HDM Expected;
+
+template <typename T>
+void CompareValues(T expected, T result, double epsilon, double threshold)
+{
+  if (std::abs(expected) >= threshold)
+  {
+    REQUIRE(result == Approx(expected).epsilon(epsilon));
+  }
+  else
+  {
+    REQUIRE(std::abs(result) <= threshold);
+  }
+}
+} // namespace
```

Comment:
remove comment :D

Reply:
This actually comes from clang formatting with the parameter FixNamespaceComments which is set by LLVM. 
Should we maybe set this in our .clang-format file?

Reply:
If this a default setting, it is fine for me